src/net/neighbors.rs

```diff
@@ -1572,8 +1936,9 @@ impl NeighborWalk {
         assert!(self.state == NeighborWalkState::ReplacedNeighborsPingFinish);
 
         let mut new_unresolved_neighbor_pings = HashMap::new();
-
-        for (nkey, mut rh) in self.unresolved_neighbor_pings.drain() {
+        let mut unresolved_neighbor_pings =
+            std::mem::replace(&mut self.unresolved_neighbor_pings, HashMap::new());
+        for (nkey, mut rh) in unresolved_neighbor_pings.drain() {
```

Comment:
What does `rh` stand for?

Reply:
Reply handle (`ReplyHandleP2P` specifically).